apps/demo/app/custom-ui/[...puckPath]/client.tsx

```diff
@@ -279,7 +279,7 @@ const CustomPuck = ({ dataKey }: { dataKey: string }) => {
 const CustomActionBar = ({ children, label }) => {
   const { appState, getPermissions, selectedItem } = usePuck();
 
-  const { debug } = getPermissions();
+  const { debug } = getPermissions({});
```

Comment:
We should default this so we don't have to provide an empty object

Reply:
fixed!